stream-chat-android-ui-common/detekt-baseline.xml

```diff
@@ -5,6 +5,7 @@
     <ID>ComplexMethod:MessageFooterVisibility.kt$ @InternalStreamChatApi public fun MessageFooterVisibility.shouldShowMessageFooter( message: Message, isLastMessageInGroup: Boolean, nextMessage: Message?, ): Boolean</ID>
     <ID>LongMethod:MessageListItemLiveData.kt$MessageListItemLiveData$ @Suppress("ComplexMethod") private fun groupMessages(messages: List&lt;Message>?, currentUserId: String): List&lt;MessageListItem></ID>
     <ID>LongParameterList:Linkify.kt$Linkify$( links: MutableList&lt;LinkSpec>, s: Spannable, pattern: Pattern, schemes: Array&lt;String>, matchFilter: Linkify.MatchFilter?, transformFilter: Linkify.TransformFilter?, )</ID>
+    <ID>MagicNumber:MessageOptionsUserReactionAlignment.kt$MessageOptionsUserReactionAlignment.BY_USER_INVERTED$3</ID>
```

Comment:
Can we fix this MagicNumber?

Reply:
The thing is it is not a MagicNumber, I would say this is a false positive, take a look in `MessageOptionsUserReactionAlignment` that is in this pr. This is reported for a single enum value in this enum and i don't understand why.

Reply:
getrekt strikes again 😆